apps/src/templates/curriculumCatalog/curriculum_catalog_card.module.scss

```diff
@@ -87,12 +87,32 @@ $container-width: 309px;
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
+  }
+}
+
+.labelTooltip {
+  opacity: 1;
+  margin-bottom: 0.75em;
+  pointer-events: none;
 
-    &:hover {
-      max-width: inherit;
-      overflow: inherit;
+  :global(.tooltip-inner) { /* stylelint-disable-line selector-pseudo-class-no-unknown */
```

Comment:
I believe this is safe because it's within the `.labelTooltip` class, which will be a unique id on the page